tests/unit/test_notifications.py

```diff
@@ -28,7 +28,7 @@ def make_workgroup(user_ids):
         ])
 
 
-class ActivityNotificationsGuineaPig(ActivityNotificationsMixin):
+class ActivityNotificationsGuineaPig(object):
```

Comment:
@ziafazal since there are no ActivityNotificationsMixin, I believe this class should also be removed. Use activity mocks where needed.